test/unit/presenters/document_export_presenter_test.rb

```diff
@@ -159,6 +159,17 @@ class DocumentExportPresenterTest < ActiveSupport::TestCase
                  result.dig(:editions, 0, :authors, 0)
   end
 
+  test "includes contacts" do
+    contact = create(:contact)
+    edition = create(:news_article,
+                     body: "[Contact:#{contact.id}]",
```

Comment:
This line probably isn't needed - what's used is the `depended_upon_contacts` array, which is populated [as per edition_dependencies_populator_test.rb](https://github.com/alphagov/whitehall/blob/master/test/unit/edition_dependencies_populator_test.rb#L5-L13). I did consider writing a test for when `depended_upon_contacts` isn't yet populated but `body` contains a contact - but don't think this is a valid scenario.

Reply:
Yeah you're right, I removed it from the test since it doesn't seem to be doing anything.